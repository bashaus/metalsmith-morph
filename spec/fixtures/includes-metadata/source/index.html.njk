{%- for file in all_files -%}
{{ file.templatePath }} => {{ file.destinationPath }}
{%- endfor %}
