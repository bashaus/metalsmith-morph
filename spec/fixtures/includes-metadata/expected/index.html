index.html.njk => index.html
